### ISSUE TITLE
scores and label

### ISSUE THREAD

**Issue by dengshuhai-cmd** [yumingj/Talk-to-Edit#12]
你们提供的训练代码中导入数据有latent_codes，labels，scores；好像labels是表示人脸图像的各种属性的表现程度（是吗？），然后其中的scores好像并没有用于计算，这个scores是什么？，最后你们可以提供一下你们训练好的属性预测模型（输入图像，输出labels的模型）吗？

**Comment by yumingj** [comment]
label是用来表示属性程度的。score是用来表示预测器对预测的程度的logits过了softmax之后的值。我们训练好的属性预测模型在我们提供的预训练模型代码里已经有了。
